### PR TITLE
env_shake: add a flag to force screen shake on clients with asw_camer…

### PR DESCRIPTION
**Pull request by jhh8**
…a_shake 0

### CODE REVIEW

**Comment by BenLubar** [comment]
asw_camera_shake is an accessibility option. not sure if it should be bypassable.

**Comment by GmaOrange** [comment]
maybe second convar to disable also bypass or something like this, so users can disable only grenade launcher annoying things.

**Comment by BenLubar** [comment]
Makes sense.

**Comment by jhh8** [comment]
another example: research center map in amber project campaign, there is a water bridge section which makes the view blurry for players. that can be disabled with mat_postprocess 0 command and those people will have a slight advantage. i personally would prefer that an effect like that wouldnt be possible to turn off, but if you guys think it should be possible since it could be too annoying or some other reason then sure, i could add another cvar for shake which would also disable the forceful shakes

File: src/game/client/view_effects.cpp [hunk 1]
```diff
@@ -685,6 +685,7 @@ screenshake_t *CViewEffects::FindLongestShake()
 }
 
 ConVar asw_camera_shake( "asw_camera_shake", "1", FCVAR_NONE, "Enable camera shakes" );
+ConVar asw_camera_shake_forced( "asw_camera_shake_forced", "1", FCVAR_NONE, "Disables forced camera shakes. Forced camera shakes are used by few maps and perform a screen shake even when asw_camera_shake is set to 0." );
```
Comment (BenLubar) [inline src/game/client/view_effects.cpp:688]: Looks like this *En*ables shake, not *Dis*ables.

**Pull request merged**